tests/lib/cli.js

```diff
@@ -282,53 +352,58 @@ describe("cli", function() {
 
     });
 
-    describe("when executing with no-eslintrc flag", function () {
-        it("should ignore a local config file", function () {
-            var exit = cli.execute("--no-eslintrc --no-ignore ./tests/fixtures/eslintrc/quotes.js");
+    describe("when executing with no-eslintrc flag", function() {
+        it("should ignore a local config file", function() {
+            var filePath = getFixturePath("eslintrc", "quotes.js");
+            var exit = cli.execute("--no-eslintrc --no-ignore " + filePath);
 
             assert.isTrue(console.log.notCalled);
             assert.equal(exit, 0);
         });
     });
 
-    describe("when executing without no-eslintrc flag", function () {
-        it("should load a local config file", function () {
-            var exit = cli.execute("--no-ignore ./tests/fixtures/eslintrc/quotes.js");
+    describe("when executing without no-eslintrc flag", function() {
+        it("should load a local config file", function() {
+            var filePath = getFixturePath("eslintrc", "quotes.js");
+            var exit = cli.execute("--no-ignore " + filePath);
 
             assert.isTrue(console.log.calledOnce);
             assert.equal(exit, 1);
         });
     });
 
-    describe("when executing without env flag", function () {
-        var files = [
-            "./tests/fixtures/globals-browser.js",
-            "./tests/fixtures/globals-node.js"
-        ];
+    describe("when executing without env flag", function() {
+        it("should not define environment-specific globals", function() {
+            var files = [
+                getFixturePath("globals-browser.js"),
+                getFixturePath("globals-node.js")
+            ];
 
-        it("should not define environment-specific globals", function () {
             cli.execute("--no-eslintrc --config ./conf/eslint.json --no-ignore " + files.join(" "));
             assert.equal(console.log.args[0][0].split("\n").length, 12);
         });
     });
 
-    describe("when executing with global flag", function () {
-        it("should default defined variables to read-only", function () {
-            var exit = cli.execute("--global baz,bat --no-ignore ./tests/fixtures/undef.js");
+    describe("when executing with global flag", function() {
+        it("should default defined variables to read-only", function() {
+            var filePath = getFixturePath("undef.js");
+            var exit = cli.execute("--global baz,bat --no-ignore " + filePath);
```

Comment:
Since all the rules are off by default, that why this file doesn't produce any errors.
Please activate the appropriate rule which looks at globals override and then this test would pass.